config/locales/en.json

```diff
@@ -525,5 +525,12 @@
 	"invite.give_code": "To finish your application, give this code and link to your doctor or nurse practioner:",
 	"invite.medical_professional_instruction": "Your doctor or nurse practioner will use this unique code to fill out a report about your medical condition(s), and how they impact your ability to work.",
 	"invite.print_button": "Print this code",
-	"invite.url": "https://cppd-medical-report-stage.herokuapp.com/en/doctor"
+	"invite.url": "https://cppd-medical-report-stage.herokuapp.com/en/doctor",
```

Comment:
All lines below this one (529-535) should be removed

Reply:
Thanks for catching! Removed.